dialogue-annotations-processor/src/test/java/com/palantir/myservice/service/MyServiceDialogueServiceFactory.java

```diff
@@ -1,278 +0,0 @@
-/*
- * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.myservice.service;
-
-import com.google.common.util.concurrent.ListenableFuture;
-import com.palantir.dialogue.BodySerDe;
-import com.palantir.dialogue.ConjureRuntime;
-import com.palantir.dialogue.Deserializer;
-import com.palantir.dialogue.DialogueServiceFactory;
-import com.palantir.dialogue.Endpoint;
-import com.palantir.dialogue.EndpointChannel;
-import com.palantir.dialogue.EndpointChannelFactory;
-import com.palantir.dialogue.HttpMethod;
-import com.palantir.dialogue.PathTemplate;
-import com.palantir.dialogue.PlainSerDe;
-import com.palantir.dialogue.Request;
-import com.palantir.dialogue.RequestBody;
-import com.palantir.dialogue.Response;
-import com.palantir.dialogue.Serializer;
-import com.palantir.dialogue.TypeMarker;
-import com.palantir.dialogue.UrlBuilder;
-import com.palantir.dialogue.annotations.Json;
-import java.util.Map;
-import java.util.Optional;
-import java.util.OptionalInt;
-import java.util.UUID;
-
-public final class MyServiceDialogueServiceFactory implements DialogueServiceFactory<MyService> {
```

Comment:
Deleting as this was an initial test just to check everything gets wired properly. In order to still keep it I'd probably need to have the MyService interface duplicated, since @DialogueClient annotation cannot be repeated. So I opted to just nuke it, and I'll enable tests when I have the implementations of the methods.